### PR TITLE
fix #78202 linkhub.icu

### LINKED ISSUE #78202: www.linkhub.icu
### Issue URL (Anti Adblock Script)
 [https://www.linkhub.icu/get/q15GyHYgxQ](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.linkhub.icu%2Fget%2Fq15GyHYgxQ)
### Comment
Username: @Jhovannino
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Firefox
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> EasyList Italy

### PR DESCRIPTION
**Pull request by wout-codes**
#78202

### CODE REVIEW

**Comment by wout-codes** [comment]
@Yuki2718 Can you write to EasyList about the BadFilter?

**Comment by Yuki2718** [comment]
@wout-codes In such case we just notify EL author and don't add filters unless the response delayed. I'll notify him.

**Comment by wout-codes** [comment]
If I ping him, he will not see my comment. I have pinged him in one of the issues about 2 months ago, and still no any fixes nor any reply.

**Comment by Yuki2718** [comment]
@wout-codes He receives so many pings everyday. Anyway I got reply, the issue will be taken care of.

**Comment by wout-codes** [comment]
So in that case, I will remove the bad filter.

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -34,7 +34,6 @@ xmoviesforyou.video,streamz.cc,drycounty.com,vidz72.com,uplod.ws,pandamovie.net,
...
....com,fx4vip.com,onlystream.tv,animesvision.biz,koreanaddict.net,iklandb.com,moaplos.com,nowagoal.xyz,bestjavs.com,thepiratebay.asia,nocensor.rest,dramahd.me,thepiratebay0.org,scat.gold,aparat.cam,clik.pw,animeblkom.net,frkmusic.me,vstplugs.com,onceddl.org,freeomovie.to,leechpremium.link,nxload.com,uii.io,short.pe,cryptohog.net,abcvideo.cc,clicknupload.co,viitodo.com,intothelink.com,snahp.it,hikarinoakariost.info,streamtape.com,animeshouse.net,drobfilx.com,skymovieshd.fit,porncomix.info,7r6.com,vldtodo.com,xyzembed379.net,zobacz.ws,vidtome.stream,legendas.dev,megasubtitles.com,haloani.ru,sukidesuost.info,ouofly.com,mixdrop.co,putload.tv,exdb.link,exdb.net,animeuniverse.it,filmisub.com,drycounty.com,guard.link,mitly.us,ouo.io,shon.xyz,bitconews.net,cda-tv.pl,picbaron.com,skidrowcodex.net,streamz.*,business-degrees.me,business-mortgage.pw,business-loan.trade,credits-loan.pw,business-credits.cc,business-mortgage.info,business-loans.pw,loan-trading.net,videogreen.xyz,thisav.com,filma24.to,playtube.ws,pornxbit.com,hitomi.la,rnbxclusive.best,onceddl.com,pornxday.com,autofaucets.org,rifurl.com,filmninja.ws,imgrock.net,shurt.pw,piratebay.live#%#//scriptlet("abort-on-property-write", "_pop")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:37]: Why did you remove it? Revert.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:37]: It's not needed.

**Review by zloyden — CHANGES_REQUESTED** [review]
Missed `@@||linkhub.icu^$generichide` for the cb platforms.

File: EnglishFilter/sections/general_extensions.txt [hunk 2]
```diff
@@ -46,7 +46,7 @@ pornxday.com,putlockerslinks.com,unblockninja.blue,baixarhentai.net,freeservice.
...
 !
 ! PopAds-b
-javhiv.com,nyafilmer.com,gotgayporn.com,letwatch.to,hqq.tv,xonline.vip,pornhd.com,thevideo.me,sextfun.com,vidup.me,imgoutlet.com,imgrock.net#%#Object.defineProperty(window, 'popns', { get: function() { return {}; } });
+linkhub.icu,javhiv.com,nyafilmer.com,gotgayporn.com,letwatch.to,hqq.tv,xonline.vip,pornhd.com,thevideo.me,sextfun.com,vidup.me,imgoutlet.com,imgrock.net#%#Object.defineProperty(window, 'popns', { get: function() { return {}; } });
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:49]: I don't see `popns` even with US IP.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:49]: Please use AdGuard for Linux, Mac or windows.

Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:49]: I have [the same](https://github.com/AdguardTeam/AdguardFilters/pull/78261#discussion_r599583029). Why did you add the domain to this rule?
@AdamWr please take a look.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:49]: Don't need him right now.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:49]: @zloyden Ryan Brown added a rule which would fix the popup, so since it's integrated with Base Filter, it's fixed as for now.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/general_extensions.txt (hunk 2 above)
Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt]: Ok.
